### DISCUSSION TITLE
Deleting old files

### DISCUSSION THREAD

**Discussion by rutledj** [nextgenhealthcare/connect#5036] [Q&A] [answered]
I'm trying to understand how to bring in the java.io.File and java.io.FileFilter packages to delete files older than x days.  Can't seem to find an example that uses these packages.

**Comment by odoodo** [comment, marked as answer]
Use [this code template](https://forums.mirthproject.io/forum/mirth-connect/support/16298-code-template-automatic-mirth-backup?p=92754#post92754) and adapt it to your needs if necessary.

**Comment by cturczynskyj** [comment]
If you simply want to know how to import and use a class like java.io.File and java.io.FileFilter in your javascript, you can do that using `importPackage(package). Example:
```
importPackage(java.io);
var myFile = new File("/path/to/file");
logger.info(myFile.toString());
```
